jujugui/static/gui/src/app/components/added-services-list/label/label.js

```diff
@@ -0,0 +1,42 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const PropTypes = require('prop-types');
+const React = require('react');
+
+class AddedServicesLabel extends React.Component {
+
+  /**
+    Calls changeState to show the readme
+  */
+  _showReadme() {
+    this.props.changeState({store: this.props.bundleURL});
+  }
+
+  /**
+    Calls changeState to show the post deployment panel.
+  */
+  _showGetStarted() {
+    this.props.changeState({postDeploymentPanel: this.props.bundleURL});
+  }
+
+  render() {
+    // A bundleURL is in the format elasticsearch-cluster/bundle/17
+    const name = this.props.bundleURL.split('/')[0].replace('-', ' ');
```

Comment:
It would be nicer to use the URL lib for this kind of transformations.
Like url.name.replace(...